strax/processing/peak_building.py

```diff
@@ -104,8 +104,8 @@ def find_peaks(hits, adc_to_pe,
 
 
 @export
-@numba.jit(nopython=True, nogil=True, cache=True)
-def store_downsampled_waveform(p, wv_buffer):
+@numba.jit(nopython=True, nogil=True, cache=False)
```

Comment:
Is there a reason why you changed the caching to False? Usually numba is quite good in bookkeeping when to recompile.